haskell-language-server.cabal

```diff
@@ -258,10 +258,7 @@ common floskell
 
 common fourmolu
   if flag(fourmolu) || flag(all-formatters)
-    hs-source-dirs: plugins/default/src
-    other-modules: Ide.Plugin.Fourmolu
-    build-depends: fourmolu                 ^>=0.3
-    cpp-options: -Dfourmolu
```

Comment:
We still need `cpp-options: -Dfourmolu` to control whether HLS wires this plugin in Main